ngraph/test/visitors/op/tanh.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gtest/gtest.h"
+
+#include "ngraph/ngraph.hpp"
+#include "ngraph/op/util/attr_types.hpp"
+#include "ngraph/opsets/opset6.hpp"
+
+#include "util/visitor.hpp"
+
+using namespace std;
+using namespace ngraph;
+using ngraph::test::NodeBuilder;
+using ngraph::test::ValueMap;
+
+TEST(attributes, tanh_op)
+{
+    using namespace opset6;
```

Comment:
If it comes to `opset` namespace in the tests, I feel less confusing if it means the opset where the particular op was introduced so `opset1` for `v0::Tanh` and if the next version of the op will be introduced then test with corresponding opset will be added.

Reply:
(https://github.com/openvinotoolkit/openvino/pull/5427/commits/c25618a2caab915914a9ca20d069e5b394959932) Well, the `using namespace` here was ok but my suggestion is that for me `opset1` would be less random than `opset6`. Anyway, it's not an issue but let's have one convention.